addon/src/main/java/com/jaimemartz/playerbalanceraddon/PlayerBalancerAddon.java

```diff
@@ -25,4 +27,17 @@ public void onDisable() {
     public PluginMessageManager getManager() {
         return manager;
     }
+
+    // This just allows people to do this programatically.
+    // I'm doing this in github.dev, pls no murder for syntax issues.
+    public void sendPlayerToServerAPI(Player player, String serverName){
+        Player target = plugin.getServer().getPlayer(player.getName());
+        if(target == null){
+            Bukkit.getServer().getConsoleSender().sendMessage("Couldn't send " + target.getName() + " to the specified Server.");
+            return;
+        }
+        target.sendMessage("Attempting to send you to server: " + serverName + "!");
+        this.getManager().sendplayer(target, serverName);
+        return;
```

Comment:
Why return? Nothing is being executed after `this.getManager$sendplayer(target, serverName);`?

Reply:
It's a default thing to add for my projects. I understand it's not necessary, IntelliJ makes that clear.

Reply:
It still should be implement like the Syntax of BGHDD and not ur own unless he asks for that. The project will just look weird with bunch of different code styles.

Reply:
> It still should be implement like the Syntax of BGHDD and not ur own unless he asks for that. The project will just look weird with bunch of different code styles.

The thing is, BGHD is my team. I manage his projects and community. BGHD is not one singular person.
Besides, the code I added, follows pretty similar syntax-wise, it's not going to break anything whilst keeping this particular line of code out of harm's way.